Sample Data/period-of-time.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix ies: <http://ies.data.gov.uk/ontology/ies4#> . 
+@prefix data: <http://data.gov.uk/testdata#> .
+@prefix iso8601: <http://iso.org/iso8601#> .
+
+data:Fred a ies:Person .
+data:FredState2 a ies:Employed ;
```

Comment:
Oops there is a mistake in the example in the standards document. There is no ```ies:Employed``` in IES4. If you could change this to a ```ies:PersonState``` I will make the change in the docs for IES 4.3

Reply:
I have made this change for IES 4.3